api/v1beta1/disruption_types.go

```diff
@@ -239,15 +240,14 @@ func (s *DisruptionSpec) Hash() (string, error) {
 	return fmt.Sprintf("%x", md5.Sum(specBytes)), nil
 }
 
-func (s *DisruptionSpec) HashNoCount() (string, error) {
-	sCopy := s.DeepCopy()
-	sCopy.Count = nil
+func (s DisruptionSpec) HashNoCount() (string, error) {
```

Comment:
Why choose to change the receiver type? Was it specifically for this function, because we might mutate the count accidentally?

Reply:
I looked for the occurrences of this struct, and changed the usage to a consistent one.

In the disruption, it's declared as NOT pointer, hence immutable. We should stay consistent everywhere and hence keep this declaration including in methods receiver.

It clarify the intent, do not modify.